client/components/card/index.jsx

```diff
@@ -79,7 +79,7 @@ let Card = React.createClass( {
 	},
 
 	render: function() {
-		var { style, title, meta, icon, iconLabel, ...other } = this.props,
+		var { style, title, meta, icon, iconLabel, iconColor, compact, ...other } = this.props,
 			cardClasses = classnames( {
 				'dops-card': true,
 				'is-compact': this.props.compact
```

Comment:
Nope, I just wanted to take it out of what will be left in `...other`. I'll change that for some better whitelisting mechanism but I wanted to have a quick overview of what's being noisy. Looks like there were not a lot of these cases :)